{"0.0.1":{"info":{"author":"","author_email":"Nathan Mendoza <nathancm@uci.edu>","bugtrack_url":null,"classifiers":["Development Status :: 3 - Alpha","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"safe-callback","package_url":"https://pypi.org/project/safe-callback/","platform":null,"project_url":"https://pypi.org/project/safe-callback/","project_urls":{"Homepage":"https://github.com/nathancmendoza/safe_callback","Issues":"https://github.com/nathancmendoza/safe_callback/issues"},"provides_extra":null,"release_url":"https://pypi.org/project/safe-callback/0.0.1/","requires_dist":null,"requires_python":">=3.10","summary":"A declarative approach to exception handling in python","version":"0.0.1","yanked":false,"yanked_reason":null},"last_serial":22398384,"urls":[{"comment_text":"","digests":{"blake2b_256":"537f048be939a7ce6bd7e0735a4b4a11f335e6f28a1255c1f6be4ee9073a5393","md5":"a53d489a4393846d60f72ab46fdf4efa","sha256":"f02ec9ac2a080d0f284484ca310416cd8de1cc8b7ae6fe2061300edf4e82cf5e"},"downloads":-1,"filename":"safe_callback-0.0.1-py3-none-any.whl","has_sig":false,"md5_digest":"a53d489a4393846d60f72ab46fdf4efa","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":2841,"upload_time":"2024-02-20T21:43:44","upload_time_iso_8601":"2024-02-20T21:43:44.288843Z","url":"https://files.pythonhosted.org/packages/53/7f/048be939a7ce6bd7e0735a4b4a11f335e6f28a1255c1f6be4ee9073a5393/safe_callback-0.0.1-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"c40bfcd06f6219d98ea4a4b7fec6a45a73c1ef24ec980cb7ac01b22c3e37d7ba","md5":"993c91e66a8f49fc5e513e41830e7231","sha256":"36f48648bb1012dbb9a32bc6c69a4907451b034e84605f8d575f2da040b19642"},"downloads":-1,"filename":"safe_callback-0.0.1.tar.gz","has_sig":false,"md5_digest":"993c91e66a8f49fc5e513e41830e7231","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":2528,"upload_time":"2024-02-20T21:43:46","upload_time_iso_8601":"2024-02-20T21:43:46.662048Z","url":"https://files.pythonhosted.org/packages/c4/0b/fcd06f6219d98ea4a4b7fec6a45a73c1ef24ec980cb7ac01b22c3e37d7ba/safe_callback-0.0.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.2":{"info":{"author":"","author_email":"Nathan Mendoza <nathancm@uci.edu>","bugtrack_url":null,"classifiers":["Development Status :: 3 - Alpha","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"safe-callback","package_url":"https://pypi.org/project/safe-callback/","platform":null,"project_url":"https://pypi.org/project/safe-callback/","project_urls":{"Homepage":"https://github.com/nathancmendoza/safe_callback","Issues":"https://github.com/nathancmendoza/safe_callback/issues"},"provides_extra":null,"release_url":"https://pypi.org/project/safe-callback/0.0.2/","requires_dist":null,"requires_python":">=3.10","summary":"A declarative approach to exception handling in python","version":"0.0.2","yanked":false,"yanked_reason":null},"last_serial":22398384,"urls":[{"comment_text":"","digests":{"blake2b_256":"5ae68fdec9e3ee6c0fcbda7a9b2f8e10fd09ba0f4b56ab76052277e9a195f9df","md5":"065cf1dda095002523a6b033ff145a25","sha256":"82836fd4ef30324be4aec7f2b777267a01e8ec38f32b12ca91e0450789a7d617"},"downloads":-1,"filename":"safe_callback-0.0.2-py3-none-any.whl","has_sig":false,"md5_digest":"065cf1dda095002523a6b033ff145a25","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":4617,"upload_time":"2024-02-24T23:18:21","upload_time_iso_8601":"2024-02-24T23:18:21.107454Z","url":"https://files.pythonhosted.org/packages/5a/e6/8fdec9e3ee6c0fcbda7a9b2f8e10fd09ba0f4b56ab76052277e9a195f9df/safe_callback-0.0.2-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"68f3ef4b335d45c99ee2bd7040ba05a477a4d7f41270eb550d6fadccc856ec13","md5":"b883e077b8d1d68ee103df90c6665ffe","sha256":"ec03cd5e67f3bf59670757350fcf0dcffa6324307fcb0ec0ed1ece9d8736d970"},"downloads":-1,"filename":"safe_callback-0.0.2.tar.gz","has_sig":false,"md5_digest":"b883e077b8d1d68ee103df90c6665ffe","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":3615,"upload_time":"2024-02-24T23:18:22","upload_time_iso_8601":"2024-02-24T23:18:22.685857Z","url":"https://files.pythonhosted.org/packages/68/f3/ef4b335d45c99ee2bd7040ba05a477a4d7f41270eb550d6fadccc856ec13/safe_callback-0.0.2.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.3":{"info":{"author":"","author_email":"Nathan Mendoza <nathancm@uci.edu>","bugtrack_url":null,"classifiers":["Development Status :: 3 - Alpha","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"","keywords":"","license":"","maintainer":"","maintainer_email":"","name":"safe-callback","package_url":"https://pypi.org/project/safe-callback/","platform":null,"project_url":"https://pypi.org/project/safe-callback/","project_urls":{"Homepage":"https://github.com/nathancmendoza/safe_callback","Issues":"https://github.com/nathancmendoza/safe_callback/issues"},"provides_extra":null,"release_url":"https://pypi.org/project/safe-callback/0.0.3/","requires_dist":null,"requires_python":">=3.10","summary":"A declarative approach to exception handling in python","version":"0.0.3","yanked":false,"yanked_reason":null},"last_serial":22398384,"urls":[{"comment_text":"","digests":{"blake2b_256":"84483ff124b10d71499ac8a29d74333378eb4d317a516c91476ecf4299aea921","md5":"7de63e1fec573c4c9dbc3fb73a521b9a","sha256":"97c098e354023673962249c09165b96e5c65e793c8edd9e1f4a2af68a47dac4c"},"downloads":-1,"filename":"safe_callback-0.0.3-py3-none-any.whl","has_sig":false,"md5_digest":"7de63e1fec573c4c9dbc3fb73a521b9a","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":2753,"upload_time":"2024-03-12T18:58:50","upload_time_iso_8601":"2024-03-12T18:58:50.718720Z","url":"https://files.pythonhosted.org/packages/84/48/3ff124b10d71499ac8a29d74333378eb4d317a516c91476ecf4299aea921/safe_callback-0.0.3-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"53aa5c298094a33173b3dc6a0756b3d07805d847ec5c04fbb43dcd11f0cbeb16","md5":"24b7fa6693db9c3a5893c7814a07db70","sha256":"113a35c36d54b889aee3cc645656d61f6a9d3981928030239767e629f3f32727"},"downloads":-1,"filename":"safe_callback-0.0.3.tar.gz","has_sig":false,"md5_digest":"24b7fa6693db9c3a5893c7814a07db70","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":2413,"upload_time":"2024-03-12T18:58:51","upload_time_iso_8601":"2024-03-12T18:58:51.809913Z","url":"https://files.pythonhosted.org/packages/53/aa/5c298094a33173b3dc6a0756b3d07805d847ec5c04fbb43dcd11f0cbeb16/safe_callback-0.0.3.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.4":{"info":{"author":null,"author_email":"Nathan Mendoza <nathancm@uci.edu>","bugtrack_url":null,"classifiers":["Development Status :: 3 - Alpha","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"safe-callback","package_url":"https://pypi.org/project/safe-callback/","platform":null,"project_url":"https://pypi.org/project/safe-callback/","project_urls":{"Homepage":"https://github.com/nathancmendoza/safe_callback","Issues":"https://github.com/nathancmendoza/safe_callback/issues"},"provides_extra":null,"release_url":"https://pypi.org/project/safe-callback/0.0.4/","requires_dist":null,"requires_python":">=3.10","summary":"A declarative approach to exception handling in python","version":"0.0.4","yanked":false,"yanked_reason":null},"last_serial":22398384,"urls":[{"comment_text":"","digests":{"blake2b_256":"caff7199687f2c4b9bc5ea268284401b4e885199b3ac4a32cae5fc13f898f7f6","md5":"f4844e004ad2b2c3b752d263d9fc19cb","sha256":"334231a869914ada46de11df10f3fd40a6743b46825a009651d531e2333dc924"},"downloads":-1,"filename":"safe_callback-0.0.4-py3-none-any.whl","has_sig":false,"md5_digest":"f4844e004ad2b2c3b752d263d9fc19cb","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":2878,"upload_time":"2024-03-19T22:40:18","upload_time_iso_8601":"2024-03-19T22:40:18.824651Z","url":"https://files.pythonhosted.org/packages/ca/ff/7199687f2c4b9bc5ea268284401b4e885199b3ac4a32cae5fc13f898f7f6/safe_callback-0.0.4-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"ad48ebecec707dc7049358ea5c99ad611503f5a9fe637ad64cb979ae6990bb46","md5":"ccccb441a5b6e066f446e9c17f98c027","sha256":"79a4e31414cde131b7a12fe58f7bb9eb1c1f43e19f05df560d983d401eb297c2"},"downloads":-1,"filename":"safe_callback-0.0.4.tar.gz","has_sig":false,"md5_digest":"ccccb441a5b6e066f446e9c17f98c027","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":2554,"upload_time":"2024-03-19T22:40:20","upload_time_iso_8601":"2024-03-19T22:40:20.527799Z","url":"https://files.pythonhosted.org/packages/ad/48/ebecec707dc7049358ea5c99ad611503f5a9fe637ad64cb979ae6990bb46/safe_callback-0.0.4.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.0.5":{"info":{"author":null,"author_email":"Nathan Mendoza <nathancm@uci.edu>","bugtrack_url":null,"classifiers":["Development Status :: 3 - Alpha","Intended Audience :: Developers","License :: OSI Approved :: MIT License","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":null,"keywords":null,"license":null,"maintainer":null,"maintainer_email":null,"name":"safe-callback","package_url":"https://pypi.org/project/safe-callback/","platform":null,"project_url":"https://pypi.org/project/safe-callback/","project_urls":{"Homepage":"https://github.com/nathancmendoza/safe_callback","Issues":"https://github.com/nathancmendoza/safe_callback/issues"},"provides_extra":null,"release_url":"https://pypi.org/project/safe-callback/0.0.5/","requires_dist":null,"requires_python":">=3.10","summary":"A declarative approach to exception handling in python","version":"0.0.5","yanked":false,"yanked_reason":null},"last_serial":22398384,"urls":[{"comment_text":"","digests":{"blake2b_256":"c223057e018ca2509d15fc3c0f91c0765592a9397568b2fe20a66ab521296686","md5":"5250b75a34aeefe451cbd7ae1d4caf5b","sha256":"69f4513bcf571a1ad7b7ac88d5f3ebafc8224b0e37b587ab9b0b1c9018dec15a"},"downloads":-1,"filename":"safe_callback-0.0.5-py3-none-any.whl","has_sig":false,"md5_digest":"5250b75a34aeefe451cbd7ae1d4caf5b","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.10","size":3879,"upload_time":"2024-03-20T00:11:15","upload_time_iso_8601":"2024-03-20T00:11:15.041750Z","url":"https://files.pythonhosted.org/packages/c2/23/057e018ca2509d15fc3c0f91c0765592a9397568b2fe20a66ab521296686/safe_callback-0.0.5-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"dfbcdd6bdb75447aa6d5b1efc6b19c29c3ac9f515e9134e67255faec48e1fa76","md5":"2fc783d35f1882d56fb212bffcf2dea7","sha256":"c938b7f3cbc72492c3537f3907c2f78ea223a7d1791dfb3447b43ee288001d0f"},"downloads":-1,"filename":"safe_callback-0.0.5.tar.gz","has_sig":false,"md5_digest":"2fc783d35f1882d56fb212bffcf2dea7","packagetype":"sdist","python_version":"source","requires_python":">=3.10","size":3607,"upload_time":"2024-03-20T00:11:16","upload_time_iso_8601":"2024-03-20T00:11:16.568463Z","url":"https://files.pythonhosted.org/packages/df/bc/dd6bdb75447aa6d5b1efc6b19c29c3ac9f515e9134e67255faec48e1fa76/safe_callback-0.0.5.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}